src/panels/config/zwave/zwave-values.ts

```diff
@@ -0,0 +1,121 @@
+import "@polymer/paper-dropdown-menu/paper-dropdown-menu";
+import "@polymer/paper-item/paper-item";
+import "@polymer/paper-listbox/paper-listbox";
+
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+
+import { haStyle } from "../../../resources/styles";
+import { HomeAssistant } from "../../../types";
+
+import "../../../components/buttons/ha-call-service-button";
+import "../../../components/ha-card";
+
+import { ZWaveValue } from "../../../data/zwave";
+
+@customElement("zwave-values")
+export class ZwaveValues extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public selectedNode: number = -1;
+  @property() public nodes: object[] = [];
+  @property() private values: ZWaveValue[] = [];
+  @property() private _selectedValue: number = -1;
+
+  protected render(): TemplateResult | void {
+    return html`
+      <div class="content">
+        <ha-card
+          header="${this.hass!.localize("ui.panel.config.zwave.values.header")}"
```

Comment:
When we're setting a dynamic value, always prefer to refer to the property over the attribute. It is faster and there won't be any conversion (in case it's not a string). Also, don't wrap dynamic values with quotes to help readability.

```suggestion
          .header=${this.hass.localize("ui.panel.config.zwave.values.header")}
```